OpenRA.Mods.Common/Traits/World/ElevatedBridgeLayer.cs

```diff
@@ -43,6 +43,7 @@ public ElevatedBridgeLayer(Actor self, ElevatedBridgeLayerInfo info)
 		public void WorldLoaded(World world, WorldRenderer wr)
 		{
 			var domainIndex = world.WorldActor.Trait<DomainIndex>();
+			var cellHeight = world.Map.Grid.Type == MapGridType.RectangularIsometric ? 724 : 512;
```

Comment:
Maybe this would be better as an extension method, to reduce duplication and make it easier to find?

Might also be worth noting how the magic 724 constant is calculated.